packages/rspack/src/stats/statsFactoryUtils.ts

```diff
@@ -159,8 +159,33 @@ export type KnownStatsModuleIssuer = {
 
 export type StatsModuleIssuer = KnownStatsModuleIssuer & Record<string, any>;
 
+export enum StatsErrorCode {
+	/**
+	 * Warning generated when either builtin `SwcJsMinimizer` or `LightningcssMinimizer` fails to minify the code.
+	 */
+	ChunkMinificationError = "ChunkMinificationError",
```

Comment:
Should we use `SwcMinificationError` and `LightningcssMinificationError`? So that users can know the error is throwed by SWC or lightmingcss, not Rspack

Reply:
We can, but I don't know if we should. For example, `ModuleBuildError` or `ModuleParseError` code does not give the exact module type. The module type is located in `moduleIdentifier`. IMO, this also applies to `ChunkMinificationError`. It might not necessary to give the exact type.

Reply:
If we can add the minimizer type to the error message, it should be good enough

![image](https://github.com/user-attachments/assets/0bbba9e9-80e5-464e-ad33-0a2b60566d84)

Reply:
This makes me thinking about the current implementation of `StatsError`. 
If need to give user the ability to create there own stats printer. Then we should add a field `pluginName` and let users define their own `StatsPrinterPlugin`.
If we only want users to rely on the current `DefaultStatsPrinterPlugin`, we just need to make it clear by adding the plugin name into the error.

Reply:
Adding the plugin name to the error seems simpler..

Reply:
An internal project in ByteDance is using `StatsError.code` and string matcher to match errors into structural object. We might just let them decide this part as most of the users don't really care about this type usage, including knowing what error belongs to what type and plugin.

Reply:
> most of the users don't really care about the type for each error, including knowing which error belongs to what type and plugin

I disagree with this. Letting normal users know that the minification bug is from SWC or lightningcss can help them debug or report issues.

Reply:
This is not what I meant. When I was talking about "most of the users don't really care about the type for each error", what I'm actually talking about is, for **ADVANCED** users, adding the plugin name to the error is not an idea choice for them. This is in addition to basic usages. For basic usages, we just need to add the plugin name.

Reply:
get 👌